src/lib/y2firstboot/clients/user.rb

```diff
@@ -71,6 +73,17 @@ def run
 
     private
 
+      # Updates user values, if needed
+      #
+      # For example, the home directory is modified to keep it on sync with the user name.
+      def update_user
+        home_path = Pathname.new(user.home || "")
+
+        return if user.home.nil? || user.name == home_path.basename.to_s
+
+        user.home = home_path.dirname.join(user.name).to_s
```

Comment:
I wonder if 

```ruby
user.home = home_path.to_s 

```

is not enough.

Reply:
Nop, `home_path` contains the old home (e.g., "/path/to/home/of/user1"), and we need to build "/path/to/home/of/user2".

Reply:
I got lost among user.**home** and user.**name** :)